Lib/fontTools/merge.py

```diff
@@ -941,6 +941,38 @@ def __delitem__(self, k):
 		del self.d[id(k)]
 
 class Merger(object):
+	"""Font merger.
+
+	This class merges multiple files into a single OpenType font, taking into
+	account complexities such as OpenType layout (``GSUB``/``GPOS``) tables and
+	cross-font metrics (e.g. ``hhea.ascent`` is set to the maximum value across
+	all the fonts).
+
+	If multiple glyphs map to the same Unicode value, subsequent glyphs are
+	renamed and a lookup in the ``locl`` feature will be created to disambiguate
+	them. For example, if the arguments are an Arabic font and a Latin font and
+	both contain a set of parentheses, the Latin glyphs will be renamed to
+	``parenleft#1`` and ``parenright#1``, and the equivalent of the following
+	feature code will be added:
+
+	.. code:: none
+
+		feature locl {
+			script latn;
+			language dflt;
+			sub [parenleft parenright] by [parenleft#1 parenright#1];
```

Comment:
this snippet of FEA code is problematic given the `#` indicates inline comments (but I guess it's ok for the time being while we address that)

Reply:
Thought of that. The `.. code:: none` turns off syntax highlighting for this block.

Reply:
I don’t find the code snippet to be adding much value IMHO, something like ”with a lookup mapping ``parenleft`` and ``parenright`` to ``parenleft#1`` and ``parenright#1``, respectively” would be enough.

Reply:
Also, if there is already a `locl` feature (with a script other than `DFLT`) the lookup will be inserted in it, so this needs to be mentioned. Also a `GSUB` table is required, merging a font with duplicate glyphs and no `GSUB` table raises an error.

Reply:
I thought the choice of script/language for the locl feature was worth noting. We can say that the user can switch between the two glyphs using locl - but it's important to know what settings you need to use to get to which glyphset.

Reply:
I guess I have aversion to using FEA as _the_ OpenType language, and prefer tool-neutral description as we shouldn’t assume everyone reading the docs can readily understand this syntax.

Reply:
OK, that's fair; I also agree that FEA is not the only language for feature code. (And certainly not the best ;-) ) Will rewrite it.